docs/02-getting-started/06-simulator.md

```diff
@@ -16,9 +16,10 @@ in-memory array of items.
 
 ## Compiling your program
 
-So now, we will compile our app and target the cloud simulator using the following command:
+So now, we return to the root of the project and compile our app and target the cloud simulator using the following command:
 
 ```sh
+cd ../../../../
```

Comment:
Nice!

We need to clean up the structure of this output directory so that the terraform file will not be buried 3,000 deep.

We have a discussion here about this topic: https://github.com/winglang/wing/issues/815